ibis/tests/backends.py

```diff
@@ -479,7 +479,7 @@ class BigQuery(UnorderedComparator, Backend, RoundAwayFromZero):
 
     @staticmethod
     def connect(data_directory: Path) -> ibis.client.Client:
-        from ibis.bigquery.tests.conftest import connect
+        from ibis.backends.bigquery.tests.conftest import connect
```

Comment:
Do you mind changing this in a separate PR? Unfortunately we can't test bigquery in PRs, so I guess this was forgotten when moving the backend. Great to have it fixed, but seems unrelated to this PR.

Reply:
sure - will try again but i had actually only made this update after getting the error (which still hasnt gone away). but yes, i will revert and handle in a separate PR.  would that be its own PR or part of an existing one?

Reply:
Also, im already up to date with master.

Reply:
> would that be its own PR or part of an existing one?

I think it's own PR. If you want to check the bigquery build, there may be more errors there, of imports that weren't updated when the backend was moved. I don't think that build was passing before introducing this, so there will be some other unrelated failing tests too.

CC: @tswast